Makefile

```diff
@@ -164,6 +165,13 @@ add-on: node_modules
 	mkdir -p $(BUILT)/$@/chrome/locale/en-US
 	cat locale/en-US/$@.properties locale/en-US/shared.properties > $(BUILT)/$@/chrome/locale/en-US/loop.properties
 
+.PHONY: chrome-manifest
+chrome-manifest: $(ADD-ON)/jar.mn
```

Comment:
No need for the .PHONY as this is a real dependency. We do however need to change this line to:

`$(BUILT)/$(ADD-ON)/chrome.manifest: $(ADD-ON)/jar,mn`

that sets up the output chrome.manifest as dependent on jar.mn.